docs/topics/learning-materials-overview.md

```diff
@@ -6,12 +6,12 @@ You can use the following materials and resources for learning Kotlin:
   * [Java to Kotlin migration guide: Strings](java-to-kotlin-idioms-strings.md) – learn how to perform typical tasks with strings in Java and Kotlin.
   * [Java to Kotlin migration guide: Collections](java-to-kotlin-collections-guide.md) — learn how to perform typical tasks with collections in Java and Kotlin.
   * [Java to Kotlin migration guide: Nullability](java-to-kotlin-nullability-guide.md) — learn how to handle nullability in Java and Kotlin.
-* [Kotlin Koans](koans.md) – complete exercises to learn the Kotlin syntax. Each exercise is created as a failing unit test and your job is to make it pass.
+* [Kotlin Koans](koans.md) – complete exercises to learn the Kotlin syntax. Each exercise is created as a failing unit test and your job is to make it pass. Recommended for developers with Java experience.
 * [Kotlin by example](https://play.kotlinlang.org/byExample/overview) – review a set of small and simple annotated examples for the Kotlin syntax.
 * [Kotlin Basics track](https://hyperskill.org/join/fromdocstoJetSalesStat?redirect=true&next=/tracks/18) – learn all the Kotlin essentials while creating working applications step by step on JetBrains Academy.
 * [Kotlin books](books.md) – find books we've reviewed and recommend for learning Kotlin.
-* [Kotlin tips](kotlin-tips.md) – look through short videos where members of the Kotlin team show how to use Kotlin in a more efficient and idiomatic way to have more fun when writing code.
-* [Advent of Code puzzles](advent-of-code.md) – learn idiomatic Kotlin and practice your language skills by completing short and fun tasks.
+* [Kotlin tips](kotlin-tips.md) – watch short videos where the Kotlin team show you how to use Kotlin in a more efficient and idiomatic way, so you can have more fun when writing code.
+* [Advent of Code puzzles](advent-of-code.md) – learn idiomatic Kotlin and test your language skills by completing short and fun tasks.
 * [Kotlin hands-on tutorials](https://play.kotlinlang.org/hands-on/overview) – complete long-form tutorials to fully grasp a technology. These tutorials guide you through a self-contained project related to a specific topic.
-* [Kotlin for Java Developers](https://www.coursera.org/learn/kotlin-for-java-developers) – course for developers with experience in Java. It shows the similarities between the two languages and focuses on what's going to be different.
-* [Kotlin documentation in the PDF format](kotlin-pdf.md) – read the whole documentation offline.
+* [Kotlin for Java Developers](https://www.coursera.org/learn/kotlin-for-java-developers) – learn the similarities and differences between Java and Kotlin.
+* [Kotlin documentation in the PDF format](kotlin-pdf.md) – read all our documentation offline.
```

Comment:
PDF doesn't contain API references :( 
So shouldn't we omit "all our" from the sentence?

Reply:
What about: "read Kotlin documents offline"? An API reference is a form of "documentation" so using "documents" instead would narrow the scope. Or we could explicitly mention that API references are not included.

Reply:
My concern was only about "all".
_read all Kotlin documentation offline, excluding API references_ (could be rephrased) also works

Reply:
Ah. I get it now! I've removed "all". I decided not to mention the exclusion of API references as it is clearly explained in kotlin-pdf.md.